src/components/Calculator.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import NumberButtons from './NumberButtons';
+import OperatorButtons from './OperandButtons';
+import { expressionStorage } from './store/store';
+import {
+  NUMBER_LIMIT,
+  ERROR_MSG,
+  CONFIRM_MSG,
+  OPERATOR,
+  OPERATOR_LIST,
+} from './constants/constant';
+import AllClearButton from './AllCearButton';
+import Screen from './Screen';
+class Calculator extends Component {
+  constructor() {
+    super();
+    window.addEventListener('beforeunload', this.confirmExist);
+  }
+  state = {
+    sum: '',
+    prevNumber: [],
+    operator: '',
+    nextNumber: [],
+  };
+
+  componentDidMount() {
+    const expression = expressionStorage.getExpression();
+    if (!expression) {
+      return;
+    }
+    const { sum, prevNumber, operator, nextNumber } = expression;
+    this.setState({ sum, prevNumber, operator, nextNumber });
```

Comment:
```suggestion
    this.setState({ ...expression });
```
여기도 spread operator로 변경해 보면 더 간결해질 것 같으네요.